### PR TITLE
Add Foxeer ReaperF745 board

### PR DESCRIPTION
**Pull request by Hwurzburg**
once we get the V3 hwdef changes up I will move this back to active board with V3 included

### CODE REVIEW

File: common/source/docs/common-foxeerf745aio.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+   -  I2C port for external compass, airspeed sensor, etc.
+   -  Micro USB port
+   -  Camera input/ VTX output
+   -  DJI Goggle connector
+
+
+   **Size and Dimensions**
+
+   - 32.5mm x 32.5mm (25.5 x 25.5mm mount pattern)
+   - 8.6g
```
Comment (andyp1per) [inline common/source/docs/common-foxeerf745aio.rst:54]: This is an AIO - we should mention the ESCs as well

Comment (Hwurzburg) [reply common/source/docs/common-foxeerf745aio.rst:54]: it mentioned in the lead and the feature list

Comment (Hwurzburg) [reply common/source/docs/common-foxeerf745aio.rst:54]: what setup...the default params?

Comment (andyp1per) [reply common/source/docs/common-foxeerf745aio.rst:54]: image

Comment (andyp1per) [reply common/source/docs/common-foxeerf745aio.rst:54]: How about something like this, right here:
```
 **ESCs**

   -  4 integrated BLHeli_S ESCs
   -  Firmware - BlueJay
   -  2S-4S input voltage, 45A burst current
   -  DShot 300/600
```

Comment (Hwurzburg) [reply common/source/docs/common-foxeerf745aio.rst:59]: okay...but 2s-6s

**Pull request merged**